src/test/java/com/avaje/tests/lib/EbeanTestCase.java

```diff
@@ -1,44 +0,0 @@
-package com.avaje.tests.lib;
-
-import com.avaje.ebeaninternal.api.SpiEbeanServer;
-import junit.framework.TestCase;
-import junit.framework.TestResult;
-
-import com.avaje.ebean.Ebean;
-import com.avaje.ebean.EbeanServer;
-import com.avaje.ebean.Transaction;
-
-/**
- * The base class for all Ebean test to get access to the Ebean server and do
- * some cleanup stuff after a test has run
- */
-public abstract class EbeanTestCase extends TestCase {
```

Comment:
Hope I did not forget to add the @Test annotation anywhere. The other issues are fixed. Also cleaned up some Checkstyle warnings where I already touched the unit-tests again.